client/my-sites/sidebar/index.jsx

```diff
@@ -183,7 +194,14 @@ export class MySitesSidebar extends Component {
 				label={ label }
 				selected={ itemLinkMatches( [ '/home' ], path ) }
 				link={ '/home' + siteSuffix }
-			/>
+			>
+				{ ! hideTaskProgress && (
+					<div className="sidebar__checklist-progress">
+						<p className="sidebar__checklist-progress-text">{ translate( 'Finish setup' ) }</p>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Finish Setup' )` *ES Score: 12*

Reply:
We'd want to stick to sentence case here, ignore the bot.